sdk/search/search-documents/src/searchIndexClient.ts

```diff
@@ -800,20 +624,8 @@ export class SearchIndexClient {
    * Retrieves the SearchClient corresponding to this SearchIndexClient
    * @param indexName - Name of the index
    * @param options - SearchClient Options
-   * @typeParam Model - An optional type that represents the documents stored in
```

Comment:
is this part of the revert intentional?

Reply:
It wasn't there before the nested select change, but it should probably stick around.